python/pyspark/sql/readwriter.py

```diff
@@ -363,7 +363,7 @@ def csv(self, path, schema=None, sep=None, encoding=None, quote=None, escape=Non
         :param timestampFormat: sets the string that indicates a timestamp format. Custom date
                                 formats follow the formats at ``java.text.SimpleDateFormat``.
                                 This applies to timestamp type. If None is set, it uses the
-                                default value, ``yyyy-MM-dd'T'HH:mm:ss.SSSZZ``.
+                                default value, ``yyyy-MM-dd'T'HH:mm:ss.SSSXXX``.
```

Comment:
Yea, they look similar instances. However, it seems the doc states 

> ... the timestamp of that moment in the current system time zone ...

> ...  using the default timezone and the default locale, ...

This probably is fine up to my knowledge.